powercmd/command.py

```diff
@@ -74,10 +74,16 @@ def short_description(self) -> str:
             return self.description.strip().split('\n', maxsplit=1)[0]
         return None
 
+    def _print_param(self, param):
```

Comment:
Nit: this function does not print anything. Maybe something like `_param_to_str`?

Also, please add a docstring to the function.